src/components/Admin/ApproveComponent.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import StoriesList from './StoriesForApproval';
+import ViewAndApproveStory from './ViewStory';
+import { Card, Row, CardHeader, CardBody, Col } from 'reactstrap';
+
+class ApproveComponent extends Component {
```

Comment:
We prolly should avoid names in form of `*SomeName*Component`. This file is inside the `components` folder, and it extends `Component` class from `React` so it's already pretty clear. Maybe something like `Admin` would be more appropriate.

Reply:
Changed to admin panel